de.dlr.sc.virsat.fdir.core/src/de/dlr/sc/virsat/fdir/core/matrix/BellmanIterator.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package de.dlr.sc.virsat.fdir.core.matrix;
+
+public class BellmanIterator extends MatrixIterator {
+	
+	private double[] baseMTTFs;
+	private double[] result = new double[probabilityDistribution.length];

```

Comment:
how does the initialisation here work?
Since probabilityDistributation is set in the constructors.